.github/ISSUE_TEMPLATE/1. yaml-error-or-warning.md

```diff
@@ -14,13 +14,13 @@ labels: 'yaml'
     the raw YAML. If it's a generic YAML Parser error, please report it here:
     https://github.com/JPinkney/yaml-ast-parser
 
-    In doubt? Read more: 
+    In doubt? Read more:
```

Comment:
These kinds of improvements are awesome!

But do them in a separate PR. I've could have merged these, but are now blocked by a comment in the actual code of this PR.